### ISSUE TITLE
latest urllib3 update results in plex not working

### ISSUE THREAD

**Issue by manuelkamp** [pannal/plex-for-kodi#68]
latest urllib3 update from official kodi addon-repo (https://kodi.tv/addons/nexus/script.module.urllib3/) results in plex not working anymore:

```2023-12-03 21:37:38.853 T:1102    error <general>: Traceback (most recent call last):
                                                   
2023-12-03 21:37:38.853 T:1102    error <general>: 
2023-12-03 21:37:38.866 T:1102    error <general>:   File "/storage/.kodi/addons/script.plexmod/lib/__init__.py", line 6, in <module>
                                                       from requests.packages.urllib3.exceptions import InsecurePlatformWarning, SNIMissingWarning
                                                   
2023-12-03 21:37:38.867 T:1102    error <general>: 
2023-12-03 21:37:38.867 T:1102    error <general>: ImportError: cannot import name 'SNIMissingWarning' from 'urllib3.exceptions' (/storage/.kodi/addons/script.module.urllib3/lib/urllib3/exceptions.py)
                                                   
2023-12-03 21:37:38.867 T:1102    error <general>: 
2023-12-03 21:37:39.130 T:1102    error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'ImportError'>
                                                   Error Contents: cannot import name 'VerifiedHTTPSConnection' from 'urllib3.connectionpool' (/storage/.kodi/addons/script.module.urllib3/lib/urllib3/connectionpool.py)
                                                   Traceback (most recent call last):
                                                     File "/storage/.kodi/addons/script.plexmod/default.py", line 2, in <module>
                                                       from lib import main
                                                     File "/storage/.kodi/addons/script.plexmod/lib/main.py", line 13, in <module>
                                                       from . import plex
                                                     File "/storage/.kodi/addons/script.plexmod/lib/plex.py", line 14, in <module>
                                                       from plexnet import plexapp, myplex, util as plexnet_util, asyncadapter, http as pnhttp
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/plexapp.py", line 402, in <module>
                                                       util.setApp(App())
                                                                   ^^^^^
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/plexapp.py", line 39, in __init__
                                                       from . import nowplayingmanager
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/nowplayingmanager.py", line 10, in <module>
                                                       from . import plexrequest
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/plexrequest.py", line 4, in <module>
                                                       from . import plexserver
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/plexserver.py", line 3, in <module>
                                                       from . import http
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/http.py", line 15, in <module>
                                                       from . import asyncadapter
                                                     File "/storage/.kodi/addons/script.plexmod/lib/_included_packages/plexnet/asyncadapter.py", line 11, in <module>
                                                       from requests.packages.urllib3.connectionpool import VerifiedHTTPSConnection
                                                   ImportError: cannot import name 'VerifiedHTTPSConnection' from 'urllib3.connectionpool' (/storage/.kodi/addons/script.module.urllib3/lib/urllib3/connectionpool.py)
                                                   -->End of Python script error report<--
```

**Comment by manuelkamp** [comment]
Workaround for those who have the same problem. Download and extract the previous version for urllib from here: https://rsync.kodi.tv/addons/nexus/script.module.urllib3/

Upload all extracted files to your raspberry running kodi into this path: /storage/.kodi/addons/script.module.urllib3/

dont forget to turn off auto updates untli this gets fixed!

**Comment by pannal** [comment]
The next version will have full urllib3 2.1.0 support. Beta is in the plex forum thread if you want to try it out.

**Comment by pannal** [comment]
Fixed with 0.6.5+rev5, thanks for reporting
